source/fundamentals/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,82 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+In this guide, we show you how to skip returned results.
+
+You can skip results using the ``skip()`` method of a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#skip(int)>`
+instance or by using the :java-docs:`Aggregates.skip() </apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html#sort(org.bson.conversions.Bson)>` 
+static factory method in an aggregation pipeline. 
+
+The ``skip()`` method takes an integer that specifies how many documents
+to omit from the beginning of the list of documents returned by your
+query. 
+
+You can use the ``skip()`` method of a ``FindIterable`` instance as follows:
+
+.. code-block:: java
+
+   collection.find().skip(2); 
+
+The ``Aggregates.skip()`` method implements the ``Bson`` interface to
+specify skip criteria as an argument.  
+
+You can use the ``Aggregates.sort()`` method as follows:
+
+.. code-block:: java
+
+   import com.mongodb.client.model.Aggregates;
+
+   collection.aggregate(Arrays.asList(Aggregates.skip(2)));
+   
+
+The example in this guide is about a paint store that sells eight
+different colors of paint.  The best colors sell quicker than the other
+colors. One day, a customer comes in and asks what the three
+best-selling colors are.  The following documents are in the
+``paint_inventory`` collection: 
+
+.. code-block:: json
+
+   { "_id": 1, "color": "red", "qty": 5 }
+   { "_id": 2, "color": "purple", "qty": 10 }
+   { "_id": 3, "color": "blue", "qty": 9 }
+   { "_id": 4, "color": "white", "qty": 6 }
+   { "_id": 5, "color": "yellow", "qty": 11 }
+   { "_id": 6, "color": "pink", "qty": 3 }
+   { "_id": 7, "color": "green", "qty": 8 }
+   { "_id": 8, "color": "orange", "qty": 7 }
+
+
+To address the scenario, you query the ``paint_inventory`` collection
+with an empty filter, sort the documents by ``qty`` field and omit the first 5
```

Comment:
Could you update all instances of numbers under 10 that aren't spelled out unless they're values or required to be in that format?

https://github.com/mongodb/docs-java/pull/67#discussion_r601987039

```suggestion
with an empty filter, sort the documents by ``qty`` field and omit the first five
```